### ISSUE TITLE
https

### ISSUE THREAD

**Issue by Antenagora** [vincentbernat/haproxy-debian-repository-wizard#14]
haproxy.list for debian buster and  haproxy 2.4 is created with http:

deb [signed-by=/usr/share/keyrings/haproxy.debian.net.gpg] http://haproxy.debian.net buster-backports-2.4 main

please consider using https: 

deb [signed-by=/usr/share/keyrings/haproxy.debian.net.gpg] https://haproxy.debian.net buster-backports-2.4 main

**Comment by vincentbernat** [comment]
On older versions of Debian, it requires to install `apt-transport-https`. Maybe in one year...

**Comment by Antenagora** [comment]
You are right, i always install apt-transport-https as one of the first steps in each debian installation.

**Comment by wwuck** [comment]
Is this ready to fix now that https is included in apt by default?

**Comment by vincentbernat** [comment]
Fixed in 779155b8d439.
